cypress/e2e/home.cy.ts

```diff
@@ -119,43 +120,19 @@ describe("Visits the home page", () => {
                 cy.url().should("include", "/terms")
             })
         })
-
-
-        describe("Visits the about page", () => {
```

Comment:
I think it's fine to remove most of this, but we should validate navigating to a link works. (especially since we generate our urls.)

Reply:
can you add an issue for adding one? would be good beginner dev opportunity